cross/armel/toolchain.cmake

```diff
@@ -16,7 +16,7 @@ add_compile_options(--sysroot=${CROSS_ROOTFS})
 set(CROSS_LINK_FLAGS "${CROSS_LINK_FLAGS} -target ${TOOLCHAIN}")
 set(CROSS_LINK_FLAGS "${CROSS_LINK_FLAGS} --sysroot=${CROSS_ROOTFS}")
 
-if("${CLI_CMAKE_RUNTIME_ID}" MATCHES "tizen.*")
```

Comment:
When build with clang, toolchain.cmake will be called several times. The first two times are called with the arguments we passed, but after that, it will be called according to the rules set in clang itself. In this case, there is no defines like CLI_CMAKE_RUNTIME_ID in its own rule, so it can not set the desired toolchain value. For the same reason, coreclr / corefx is also using environment value. However, in the environment where gcc-arm-linux-gnueabi is installed, even if there is no toolchain value for Tizen, the build is done, so I uploaded the previous patch. However, the docker currently used by ci does not have the gcc library and is not normally built.